src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -212,9 +227,27 @@ angular.module("ovh-angular-mondial-relay")
                     this.loading.search = true;
                     this.ngModel = null;
                     this.foundRelays = [];
+                    var filt = filter;
```

Comment:
What's a `filt` ? What's the point of this variable ?

Reply:
To avoid modifying the parameter itself

Reply:
No I meant, `filt` is not a word, it should probably be named otherwise 😄.

And please, don't name it `otherwise` 😅